lib/src/components/NxTreeView/NxTreeView.tsx

```diff
@@ -50,7 +50,7 @@ const NxTreeView: FunctionComponent<Props> =
           triggerTooltipProps ? (
             // div necessary to avoid error message when tooltip is on disabled button
             <NxTooltip { ...triggerTooltipProps } >
-              <div>
+              <div className="nx-tree-view__tooltip">
```

Comment:
I'd rather a better name for this – it isn't the tooltip itself.  Maybe `nx-tree-view__tooltip-wrapper`?

Reply:
> I'd rather a better name for this – it isn't the tooltip itself. Maybe `nx-tree-view__tooltip-wrapper`?

https://github.com/sonatype/sonatype-react-shared-components/pull/408/commits/9b3afaf8deb50a3720d690413af3f00a0ddf7e3c